src/UI/Component/Listing/Descriptive.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+
+namespace ILIAS\UI\Component\Listing;
+
+/**
+ * Interface Descriptive
+ * @package ILIAS\UI\Component\Listing
+ */
+interface Descriptive extends Listing {
+
+	/**
+	 * Sets a key value pair as items for the list. Key is used as title and value as content.
+	 * @param string[] $items string => Component | string
```

Comment:
`string[]` imho should be `(string|Component)[]` or just `array`. Canonical types are listed here: [phpDocumentor/Types](https://www.phpdoc.org/docs/latest/guides/types.html)